docs/static/settings/dlq-settings.asciidoc

```diff
@@ -0,0 +1,11 @@
+// The content for these settings is single sourced from attributes-ls.asciidoc
```

Comment:
Note to self @karenzone: `Add dead_letter_queue.retain.age` for both logstash.yml and pipelines.yml